pycytominer/operations/variance_threshold.py

```diff
@@ -1,6 +1,6 @@
-"""
-Remove variables with near-zero variance.
-Modified from caret::nearZeroVar()
+"""Remove variables with near-zero variance.
+
+Modified from caret::nearZeroVar().
```

Comment:
Is this accurate? Mostly trying to differentiate from text vs code here. If true, would it be worth crediting the authors of `caret` somehow?
```suggestion
Modified from `caret::nearZeroVar()` in R.
```